rest/data/aliases_test.go

```diff
@@ -68,46 +68,48 @@ func (a *AliasSuite) SetupTest() {
 }
 
 func (a *AliasSuite) TestFindProjectAliases() {
-	found, err := a.sc.FindProjectAliases("project_id", "")
+	found, err := a.sc.FindProjectAliases("project_id", "", nil)
 	a.NoError(err)
 	a.Len(found, 3)
 
-	found, err = a.sc.FindProjectAliases("project_id", "repo_id")
+	found, err = a.sc.FindProjectAliases("project_id", "repo_id", nil)
 	a.NoError(err)
 	a.Len(found, 3) // ignore repo
 
-	found, err = a.sc.FindProjectAliases("non-existent", "")
+	found, err = a.sc.FindProjectAliases("non-existent", "", nil)
 	a.NoError(err)
 	a.Len(found, 0)
 
-	found, err = a.sc.FindProjectAliases("non-existent", "repo_id")
+	found, err = a.sc.FindProjectAliases("non-existent", "repo_id", nil)
 	a.NoError(err)
 	a.Len(found, 1) // from repo
 
-	found, err = a.sc.FindProjectAliases("", "repo_id")
+	found, err = a.sc.FindProjectAliases("", "repo_id", nil)
 	a.NoError(err)
 	a.Len(found, 1)
+
+	// TODO: add test
```

Comment:
did you need to do this now?

Reply:
Yes but I'm going to hold off depending on what we decide with the function signature